src/SessionManager.php

```diff
@@ -97,24 +97,7 @@ public function __construct(
      */
     public function sessionExists()
     {
-        if (session_status() === PHP_SESSION_ACTIVE) {
-            return true;
-        }
-
-        /**
-         * @var string|false $sid
-         */
-        $sid = defined('SID') ? constant('SID') : false;
```

Comment:
Removal of this might cause problems, see: https://github.com/laminas/laminas-session/pull/98

Can't remember exactly what the problem is, but it's of the legacy BC variety if memory serves. We may have to keep this in until 3.x